e2e_playwright/lazy_loaded_modules.py

```diff
@@ -37,8 +37,7 @@
     "rich",
     "tenacity",
     # toml is automatically loaded if there is a secret.toml, config.toml or
-    # a local credentials.toml file.
-    "toml",
+    # a local credentials.toml file. So, we cannot test this here.
```

Comment:
I don't understand why "toml" is not in the list of `lazy_loaded modules` anymore, do we use it now unconditionally now on streamlit app bootstrap, probably better to clarify in the comment, because it is not obvious. 
Or maybe even remove the comment completly since anyway now toml is not in the list.

Reply:
Yea toml gets loaded if secrets.toml is included, since the tests include a secrets.toml in the default, every e2e test now receives it and loads with `toml`. I believe this has changed because we want secrets to work in bare execution.